.github/workflows/windows.yml

```diff
@@ -663,10 +663,12 @@ jobs:
     defaults:
       run:
         shell: pwsh
-    runs-on: windows-latest
+    runs-on: windows-latest-8-cores
```

Comment:
Why not 4-cores like for Linux? Is it more profitable?

Reply:
there is no 4-cores runners on windows (8 and more only)